frontend/src/lib/util/util.ts

```diff
@@ -94,7 +95,10 @@ export function getEndpoint() {
 		return '/localzeno';
 	}
 	if (env.PUBLIC_BACKEND_ENDPOINT == 'http://zeno-backend:8000') {
```

Comment:
Should we remove the localzeno stuff? Seems like this would be needed there as well otherwise.